tests/materialize/materialize-postgres-rc/setup.sh

```diff
@@ -3,12 +3,13 @@
 set -e
 
 # Ensure canonical Postgres environment variables are set.
-export PGHOST="${PGHOST:=localhost}"
+export PGHOST="${PGHOST:=materialize-postgres-rc-postgres-1.flow-test}"
 export PGPORT="${PGPORT:=5432}"
-export PGDATABASE="${PGDATABASE:=postgres}"
-export PGPASSWORD="${PGPASSWORD:=postgres}"
-export PGUSER="${PGUSER:=postgres}"
+export PGDATABASE="${PGDATABASE:=flow}"
+export PGPASSWORD="${PGPASSWORD:=flow}"
+export PGUSER="${PGUSER:=flow}"
 
+docker compose -f materialize-postgres-rc/docker-compose.yaml up --detach
```

Comment:
Any reason to have this here as opposed to setting it up in github action?

Reply:
I just thought it makes sense that each setup.sh file sets up its environment as much as possible (although sometimes earlier CI steps need the service to be up), and in case of running tests locally, it makes things simpler, you just have to do
```
CONNECTOR=x VERSION=y ./tests/run.sh
```

without worrying about what services need to be run, etc.